plugins/dbgate-plugin-redis/src/backend/driver.js

```diff
@@ -91,9 +91,11 @@ const driver = {
       if (_.isNumber(database)) db = database;
       pool = new Redis({
         host: server,
+        username: user,
         port,
         password,
         db,
+        tls: {},
```

Comment:
Why this is neccessary? This line breaks connection to redis locally (Without user it works)